pom.xml

```diff
@@ -624,7 +624,7 @@
         <dependency>
             <groupId>org.onebusaway</groupId>
             <artifactId>onebusaway-gtfs</artifactId>
-            <version>1.3.95</version>
+            <version>1.3.96-SNAPSHOT</version>
```

Comment:
Seems that this cannot work.

Reply:
I see the CI here doesn't pull snapshots, so if you need to see the build succeed (which I can understand), I can probably get GTFS-mods released... again, let me know your preferences here.

Reply:
I'd strongly prefer to use a release version rather than a snapshot.

Reply:
So if you could make the release happen, that'd be great.

Reply:
Done